.github/workflows/release-discover-branches.yaml

```diff
@@ -30,6 +29,13 @@ jobs:
         with:
           go-version-file: ./src/github.com/openshift-knative/hack/go.mod
 
+      - uses: tektoncd/actions/setup-tektoncd-cli@main
+        with:
+          version: latest
+
+      - name: Install Kustomize
+        run: GO111MODULE=on go install sigs.k8s.io/kustomize/kustomize/v5@latest
```

Comment:
nit: kustomize is already in ubuntu-latest runner image installed (https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#tools)

Reply:
I think having a specific version helps?